benchadapt/python/README.md

```diff
@@ -1,3 +1,132 @@
 # Python {benchadapt}
 
-A small python package with utilities for running Conbench benchmarks.
+A small python package with utilities for getting benchmark results into a Conbench server.
```

Comment:
I wanted to mention this before; many times. I think the term "server" is not ideal

I can see that this is a bit of a cultural thing in computing terminology, but "server" here sounds a bit naive to me.

Conbench is a web application in the most traditional sense, and one component thereof is the WSGI server (gunicorn). There also is an HTTP server (technically a reverse proxy) in front of the Conbench web application. 

We can say things like "into Conbench". Or to "a Conbench instance". Or to "a Conbench deployment". Or "to the Conbench API".

Reply:
> small python package with utilities

Can you add here whether this is a CLI? Or a set of CLIs? Or a library and CLI?

Reply:
I've been trying to avoid "into Conbench" because we have a lot of things named Conbench (the runner CLI, the runner, various classes, the webapp, etc), and I'm always referring to a particular one. "Conbench instance" sounds like a class instance to me (of which there are a handful named `Conbench`). "Conbench API" is probably most apt here.